test/autorest/generated/xmlgroup/internal/xmlgroup/models.go

```diff
@@ -390,14 +511,15 @@ type XMLGetEmptyWrappedListsResponse struct {
 // XMLGetHeadersResponse contains the response from method XML.GetHeaders.
 type XMLGetHeadersResponse struct {
 	// CustomHeader contains the information returned from the CustomHeader header response.
-	CustomHeader *string `xml:"CustomHeader"`
+	CustomHeader *string
```

Comment:
is it right that this doesn't have the XML tag anymore?

Reply:
Yes, this value comes from an HTTP response header so no tag is required.